viewer/packages/rendering/src/glsl/pointcloud/edl.glsl

```diff
@@ -0,0 +1,37 @@
+// Based on Potree shader and https://github.com/CloudCompare/CloudCompare/blob/master/plugins/core/GL/qEDL/shaders/EDL/edl_shade.frag
+// Function calculating obscurance of specified fragment.
+// Passed colorTexture is assumed to have logarithmic depth value written to alpha channel.
+
+#define MAX_FLOAT 3.402823466e+38;
+
+vec2 calculateObscurance(float depth, float screenWidth, float screenHeight, vec2 neighbours[NEIGHBOUR_COUNT],
+	float radius, vec2 vUv, sampler2D depthTexture, sampler2D colorTexture) {
+
+	vec2 uvRadius = radius / vec2(screenWidth, screenHeight);
+
+	float sum = 0.0;
+	float minNeighbourDepth = MAX_FLOAT;
+	vec2 minNeighbourUv = vec2(0.0, 0.0);
+
+	for(int i = 0; i < NEIGHBOUR_COUNT; i++){
```

Comment:
Is `NEIGHBOUR_COUNT` mutable from user space? If not, perhaps we should just hard code it into this shader. Also, we have had performance issues previously because some drivers such at loop unrolling, so we might want to concider manually unrolling this loop.

Reply:
This constant is used both in normalize.frag and edl.glsl.. I think we should have it as a `#define` parameter to keep them consistent

Reply:
For the unrolling tip - we'll test on iOS and get back to this

Reply:
Confirmed to work well on iOS